MPSGameLift/Code/Source/MatchmakingSystemComponent.cpp

```diff
@@ -264,10 +265,10 @@ namespace MPSGameLift
         AZStd::string httpLatenciesParam;
         for (auto const& [region, latencyMs] : regionalLatencies)
         {
-            httpLatenciesParam += AZStd::string::format("%s_%lld ", region.c_str(), latencyMs.count());
+            httpLatenciesParam += AZStd::string::format("%s_%u_", region.c_str(), aznumeric_cast<uint32_t>(latencyMs.count()));
```

Comment:
There is a trailing `_` in the string, and i think we should use the platform-agnostic `PRIi64` instead
```suggestion
            httpLatenciesParam += AZStd::string::format("%s_%" PRIi64 , region.c_str(), aznumeric_cast<uint32_t>(latencyMs.count()));
```

Reply:
Fixed!